### DISCUSSION TITLE
yalmip mosek

### DISCUSSION THREAD

**Discussion by Ouyang-maxzx** [yalmip/YALMIP#1105] [Problems using YALMIP]
Dear Prof Johon
        how to call mosek to solve problem ,when call mosek ''ops=sdpsettings('solver','mosek',verbose',2,'debug','1','usex0',0);'',it will calling "mosekopt", one or more output parameters were not assigned a value.

**Comment by johanlofberg** [comment]
I have no idea what you are saying or asking
